symphony-bdk-spring/symphony-bdk-core-spring-boot-starter/src/main/java/com/symphony/bdk/spring/annotation/SlashAnnotationProcessor.java

```diff
@@ -43,6 +48,13 @@
       "com.symphony.bdk.core."
   };
 
+  private static final ParameterNameDiscoverer PARAMETER_NAME_DISCOVERER = new DefaultParameterNameDiscoverer();
+
+  /**
+   * Key is the annotated slash method, value is the map of (parameter name, index)
+   */
+  private static final Map<Method, Map<String, Integer>> METHOD_TO_ARGUMENT_INDEXES = new HashMap<>();
```

Comment:
do we use it as a cache? I would expect to find in the code something where we try to read an entry and if we don't find it we initialize, here it looks like it is used as a regular field (or maybe I missed something)

Reply:
Yes and it is initialized line 215. I will update the implementation to make it clearer.

Reply:
Well, it seems we don't really need a cache. The DefaultNameParameterNameDiscoverer already caches some of the results (see LocalVariableTableParameterNameDiscoverer in SB)